types/beaver-logger/types.d.ts

```diff
@@ -0,0 +1,5 @@
+export const TYPES = true;
```

Comment:
```suggestion
export const TYPES = true;
```
what are those? 
https://github.com/krakenjs/beaver-logger/blob/master/src/types.js

Reply:
@ispedals: ping?